### ISSUE TITLE
[Enhancement] Improve Accessibility

### ISSUE THREAD

**Issue by Aman-Codes** [meshery/meshery.io#527]
**Description**
Need to fix the below accessibility issues in the meshery.io website
- Insufficient contrast ratio affecting text readability
- Non-consecutive heading level tag
- Missing labels in the Meshery Community Newsletter form
- Improve screen reader experience

**Expected Behavior**
No accessibility issues and good screen reader experience

**Screenshots**

Accessibility report of meshery.io https://wave.webaim.org/report#/https://meshery.io/
![image](https://user-images.githubusercontent.com/54680709/130942125-d742a678-33c5-4b82-8354-74d9d3930304.png)

---

### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

**Comment by leecalcote** [comment]
@Aman-Codes, I'd love to see issues like this opened on https://github.com/meshery/meshery (for meshery.io).

**Comment by contracamilo** [comment]
@Aman-Codes  if you want I cant work on this issue.  :)

**Comment by yahyakhn** [comment]
Is this issue available? i can work on it.
